R/checks.R

```diff
@@ -383,3 +426,24 @@ check_expected_outputs = function(tool_name="battenberg",seq_type_filter="genome
 }
 
 
+#' @title Check Config Value.
+#' 
+#' @description Check the existence of a specific config key. 
+#' The function will notify the user if no such key exists.
```

Comment:
"will notify the user and end the program if..."

Reply:
Nice, I have updated the docs for this function.